docs/Makefile.am

```diff
@@ -125,17 +125,75 @@ SPELLCHECK_SRC = $(ALL_TXT_SRC) ../README ../INSTALL.nut ../UPGRADING  ../NEWS \
 	../TODO ../scripts/ufw/README ../scripts/augeas/README ../lib/README \
 	../tools/nut-scanner/README
 
+# Note: de-facto our documentation is beyond ASCII (at least in names of
+# international committers). The grep tests below look if the aspell output
+# contained something other than the OK lines (tagged with asterisk) and
+# aspell's version (tagged with @) and if it did - those lines must be the
+# spellcheck complaints. Empty OUT is ok.
+# We also must indent the input, because certain piped-in characters are
+# interpreted as commands, and seems this feature can not be turned off.
+# See also http://aspell.net/man-html/Through-A-Pipe.html
+# TODO: Is "grep -a" or "grep -b" (treat input as ascii/bin) portable enough?
+# Is "egrep == grep -E" always valid? (maybe all a job for configure.ac)
+# TODO: Define a custom language in-repo or during build, that is english
```

Comment:
Custom language - probably not anymore, at least not until we hit a platform with sufficiently different or old aspell.

Concerns of portable `egrep` and `-a/-b` flags - these may still be valid (buildbots might show what works where). If the issues do pop up, the flag may be removed at expense of warnings that unexpected binary input was detected - and those might need to be grepped out so as to not pollute logged reports and/or interpretation of aspell's "$OUT".